jsapp/js/components/permissions/userPermissionRow.component.tsx

```diff
@@ -163,27 +164,29 @@ export default class UserPermissionRow extends React.Component<
             <bem.UserRow__perms>{t('Pending owner')}</bem.UserRow__perms>
           )}
 
-          {!this.props.isUserOwner && !this.props.isPendingOwner && (
-            <React.Fragment>
-              {this.renderPermissions(this.props.permissions)}
-
-              <bem.Button m='icon' onClick={this.toggleEditForm.bind(this)}>
-                {this.state.isEditFormVisible && (
-                  <i className='k-icon k-icon-close' />
-                )}
-                {!this.state.isEditFormVisible && (
-                  <i className='k-icon k-icon-edit' />
-                )}
-              </bem.Button>
-
-              <bem.Button
-                m='icon'
-                onClick={this.showRemovePermissionsPrompt.bind(this)}
-              >
-                <i className='k-icon k-icon-trash' />
-              </bem.Button>
-            </React.Fragment>
-          )}
+          {this.props.displayControls &&
+            !this.props.isUserOwner &&
+            !this.props.isPendingOwner && (
+              <React.Fragment>
+                {this.renderPermissions(this.props.permissions)}
```

Comment:
The use of `displayControls` needs to be changed a bit. That `this.renderPermissions` is displaying a list of permissions given user has. We should probably hide/disable just the buttons below

Reply:
Good catch, thank you!